src/Lynx/List.purs

```diff
@@ -67,14 +69,17 @@ type Column
 -- | back into the logic of what a "view" really does.
 type RenderView
   = { columns :: Array RenderColumn
+    , rows :: Array (Lynx.Form.Page Lynx.Expr.ExprType)
     }
 
 type RenderColumn
   = { name :: Lynx.Expr.ExprType
+    , value :: Lynx.Expr.Key
     }
 
 type Input
   = { columns :: Array Column
+    , rows :: Array (Lynx.Form.Page Lynx.Expr.ExprType)
```

Comment:
Can we assume that we're getting `Page ExprType`? I think the `GET` endpoint will return `Page Expr`, since that's what the form will expect, and may even be what we'll expect here. I guess this is one of those things we can kick down the road, but I do think we'll end up evaluating each row before rendering it.

Reply:
Hmm, I'm not sure. This is what I was thinking: we can only create orders when they have all the values, those values would have to be someone typing in strings that we're evaluating, so we'd be storing the values as `ExprType` not parsing them to `Expr`. Is that not how it's going to work?

Reply:
> we can only create orders when they have all the values

Only when we have `required :: Boolean true` on all fields. But Spencer doesn't want us to worry about validation for now, so we're saving everything. Also, if every field has a `default :: Just _`, then we have values anyway. 

> so we'd be storing the values as `ExprType` not parsing them to `Expr`. Is that not how it's going to work?

I think we should always store values as `ExprType`, even for `Page Expr`. I don't think a value of `InputSource Expr` ever makes sense, as users can't input expressions (maybe in the Form Builder form it'd make sense). 

But returning `Page Expr` instead of `Page ExprType` is necessary, at least for the form, because it needs to know what the expressions are in order to evaluate changes to the form. At the least we'd need to send both, but that seems redundant because we can get one from the other.

For the list view it's less clear that `Page Expr` is necessary, but I argue that it is, because evaluation isn't pure, there are HTTP lookups that can happen, or date validations that might happen, which means the result of evaluating now might be different from the result of evaluating when the Order was saved. I guess which result is more correct is debatable, but I think the newer one.

Reply:
I see. It's important to note that whole thing isn't accurate anyway. We should be working with an Order, not a `Page _` (whatever that is in the OMS parlance). I think no matter what we put in here, it'll be an approximation of what the real thing is until we get to that.